proto/exocore/oracle/info.proto

```diff
@@ -17,14 +17,16 @@ message Chain{
 message Token{
   // token name
   string name = 1;
-  //id refer to chainList's index
+  // id refer to chainList's index
   uint64 chain_id = 2 [(gogoproto.customname) = "ChainID"];
-  //if any, like erc20 tokens
+  // if any, like erc20 tokens
   string contract_address = 3;
   // decimal of token price
   int32 decimal = 4;
-  //set false when we stop official price oracle service for a specified token
+  // set false when we stop official price oracle service for a specified token
   bool active = 5;
+  // refer to assetID from assets module if exists
+  string asset_id = 6 [(gogoproto.customname) = "AssetID"];
```

Comment:
This is unused as of now, correct?

Reply:
Yes, operator use assetID to query price. Set this field to map from assetID to tokenID to keep oracle module independently.

Reply:
Why use two different identifiers ? Do you intend on merging them later so that everything is following `assetID` ?

Reply:
Oracle module need to keep tokenInfos for history, since tokenFeeders refer to tokenID. And assets module currently seems able to modify assetsInfo. It's possible to merge these two, but need to do some adjustments with assets module